### PR TITLE
Feat/add stylelint config

### PR DESCRIPTION
**Pull request by MKMonkeyCat**
# Description

feat stylelint config

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by SiongSng — CHANGES_REQUESTED** [review]
There is a problem with the animation of the home page

https://user-images.githubusercontent.com/48402225/170826684-b3962b3b-60fe-4ee6-ab7f-fccbe2f882ab.mp4

**Comment by MKMonkeyCat** [comment]
done

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
   "recommendations": [
-    "Vue.volar",
+    "vue.volar",
```
Comment (SiongSng) [inline .vscode/extensions.json]: Please do not modify the ID of the extension

**Review by SiongSng — CHANGES_REQUESTED** [review]

File: .vscode/extensions.json [hunk 2]
```diff
@@ -3,9 +3,10 @@
     "Vue.volar",
     "esbenp.prettier-vscode",
     "Vue.vscode-typescript-vue-plugin",
-    "Lokalise.i18n-ally",
+    "lokalise.i18n-ally",
```
Comment (SiongSng) [inline .vscode/extensions.json:6]: Please do not modify the ID of the extension

**Review by SiongSng — CHANGES_REQUESTED** [review]

File: .vscode/extensions.json (hunk 2 above)
Comment (a3510377) [reply .vscode/extensions.json:6]: 這才是對的Emm
lokalise.i18n-ally

**Comment by MKMonkeyCat** [comment]
done

**Review by SiongSng — APPROVED** [review]
LGTM!

**Pull request merged**